src/Analysis/GWASV2/Components/SelectCohort/SelectCohortDropDown.stories.jsx

```diff
@@ -0,0 +1,258 @@
+import React, { useReducer } from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { rest } from 'msw';
+import SelectCohortDropDown from './SelectCohortDropDown';
+import reducer from '../../Shared/StateManagement/reducer';
+import { Space } from 'antd';
+import { SourceContextProvider } from '../../Shared/Source';
+import ACTIONS from '../../Shared/StateManagement/Actions';
+import './SelectCohort.css';
+import '../../../GWASV2/GWASV2.css';
+import '../../../GWASUIApp/GWASUIApp.css';
+
+export default {
+  title: 'Tests3/GWASV2/CohortSelect/SelectCohortDropDown',
+  component: SelectCohortDropDown,
+};
+
+// useful examples: https://github.com/mswjs/msw-storybook-addon/tree/main/packages/docs/src/demos/react-query
+
+const mockedQueryClient = new QueryClient({
+  defaultOptions: {
+    queries: { retry: false },
+  },
+});
+
+const MockTemplate = () => {
+  const initialState = {
+    outcome: {},
+    selectedStudyPopulationCohort: {
+      cohort_definition_id: 400,
+      cohort_name: 'Test cohortC - Large',
+      size: 200000,
+    },
+    covariates: [],
+    imputationScore: 0.3,
+    mafThreshold: 0.01,
+    numOfPC: 3,
+    gwasName: '',
+    selectedHare: {
+      concept_value: '',
+    },
+    currentStep: 0,
+  };
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  const handleStudyPopulationSelect = (selectedRow) => {
+    dispatch({
```

Comment:
This confused me because I don't think this method is related to this component. Maybe change this method to perform a state update to {setFirstPopulation} or {setSecondPopulation} similar to its usage in the application?

Reply:
good catch. Removed copy+paste code